tests/unittests/conftest.py

```diff
@@ -0,0 +1,360 @@
+# Copyright (C) 2023 Red Hat, Inc.
```

Comment:
```suggestion
# Copyright (C) 2024 Red Hat, Inc.
```

Reply:
If this change is accepted then we should probably make the same changes to the other files in this pull request